cmd/flags.go

```diff
@@ -150,6 +155,30 @@ func initEthereumFlags(cmd *cobra.Command, cfg *config.Config) {
 	)
 }
 
+// Initialize flags for Electrs configuration.
+func initElectrsFlags(cmd *cobra.Command, cfg *config.Config) {
```

Comment:
Similarly to `config.BitcoinElectrum`, if we do `[bitcoin.electrum]` in the config, then this one should probably be:
```suggestion
func initBitcoinElectrumFlags(cmd *cobra.Command, cfg *config.Config) {
```

Reply:
Done in b26f5297258e43a7f57047a0157e5f3b111adab5.